library/azul-zulu

```diff
@@ -0,0 +1,98 @@
+Maintainers: azul_openjdk@azul.com
```

Comment:
Build is failing because it is expecting a name. Since this seems like a shared mailbox you can do something like this: https://github.com/docker-library/official-images/blob/master/library/amazoncorretto#L1

Reply:
... but please do include at least one human GitHub handle (or the handle of a bot whose email the whole team will receive that we can @-reference from non-Azul orgs)